package.json

```diff
@@ -20,7 +20,7 @@
     "postinstall": "NODE_ENV=production npm run production"
   },
   "dependencies": {
-    "eclipsefdn-hugo-solstice-theme": "0.0.150"
```

Comment:
That is quite the jump!  We will want to make sure to properly test the site to ensure we don't introduce any regressions.

Reply:
I tested it and it seems fine! I did find an unrelated bug though: #547